.changeset/config.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://unpkg.com/@changesets/config@3.0.3/schema.json",
-  "baseBranch": "origin/v0.2.x",
+  "baseBranch": "v0.2.x",
```

Comment:
delete the `origin/` to avoid confusion when working with many remote servers specified in git